core/chains/evm/gas/chainoracles/l1_gas_price_oracle.go

```diff
@@ -0,0 +1,176 @@
+package chainoracles
```

Comment:
nit: Just wondering, if this name is appropriate.
This seems to be a package for Rollups.
Could rollups be a better name?